src/api/CartApi.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import { toastText } from "../utils/toast";
+
+export const AddToCart = async (item, cartId, dispatch) => {
+  if (cartId === "") {
+    const cartResponse = await axios.post(
+      "https://serene-lowlands-13656.herokuapp.com/carts",
+      {
+        productsArray: { _id: item._id, productId: item._id, quantity: 1 },
+      }
+    );
+
+    dispatch({
+      type: "FIND_CARTID",
+      payload: cartResponse.data.CartData._id,
+    });
+    dispatch({
+      type: "ADDTOCART",
```

Comment:
use consistent naming convention. since you've written dispatch in `DEMO_DEMO` format. would be good to write `ADDTOCART` as `ADD_TO_CART`